### DISCUSSION TITLE
Additional pattern to translate

### DISCUSSION THREAD

**Discussion by dferb** [orbikm/mkdocs-ezlinks-plugin#37] [Ideas]
Hi, 

First, thank you for a great plugin. Saves quite some time with the way I work. One suggestion though:

I use mkdocs with option: use_directory_urls: true (which is also default). So the URL'g generated are in form `something/file/#anchor`

So if the plugin could support links in form `[Text](file/#anchor)` - so with additional `/` this would make creating link a simple copy paste. Not really sure if it makes sense in the wider picture though.

**Comment by orbikm** [comment]
Hello,

> First, thank you for a great plugin. Saves quite some time with the way I work.

Thanks a lot! I don't get a ton of direct feedback, so this is great to hear!

I could definitely see this being a behavior change of the plugin based on whether or not `use_directory_urls` is true / false. I will add an issue to track it soon.

Thank you for the suggestion!
